resources/views/profile/two-factor-authentication-form.blade.php

```diff
@@ -91,7 +91,8 @@ class="button-secondary w-full sm:w-auto"
             <span class="header-4">@lang('fortify::pages.user-settings.2fa_title')</span>
             <span class="mt-4">@lang('fortify::pages.user-settings.2fa_description')</span>
 
-            <div class="flex mt-8">
+            <div class="mt-4 sm:mt-8 flex flex-col items-center space-y-4 sm:items-start sm:flex-row sm:space-y-0 sm:space-x-6">
+                <img src="{{ asset('/images/profile/2fa.svg') }}" class="w-24 h-24" alt="">
```

Comment:
if I use `x-ark-icon` the image not load..

Reply:
yeah for the `img` ones you can leave it, it's purely when you use `@svg` it's better to use `x-ark-icon` instead. The `img` tags are usually used when it's for images that we need to be able to overwrite or are expected to exist in the project but are not part of fortify itself